azerothcore/azerothcore-wotlk
# fix(DB/Misc): Fix gossip conditions for 'Altar of Suntara' and add ac…

…tivation delay

* Should only offer the option to summon Lathoric if the player has the quest Rise, Obsidion in progress
* Implement missing object activation delay, should only be usable again after a while once the gossip is selected



## Changes Proposed:
* Should only offer the option to summon Lathoric if the player has the quest Rise, Obsidion in progress
* Implement missing object activation delay, should only be usable again after a while once the gossip is selected


## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


.go c 5569 to port there. Verify if you can start the quest directly at the altar without even having the quest.

Start the quest [Rise, Obsidion!] in the Slag Pits in Searing Gorge. You may need the pre-req, which is "Release Them" - .q add 3372 and .additem [mysterious artifact], handin at the altar.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
